tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
```

Comment:
marking spaces or squares? What terminology you use here should be followed throughout the document. In line 15 you are using square.